packages/web/src/pages/sign-up-page/pages/SelectArtistsPage.tsx

```diff
@@ -198,13 +205,13 @@ export const SelectArtistsPage = () => {
                     wrap='wrap'
                     justifyContent='center'
                   >
-                    {isLoading ? (
-                      <LoadingSpinner />
-                    ) : (
-                      artists?.map((user) => (
-                        <FollowArtistTile key={user.user_id} user={user} />
-                      ))
-                    )}
+                    {isLoading
+                      ? range(9).map((index) => (
```

Comment:
any sense in doing `artists?.length > 0 ? artists.length : 9`?

Reply:
hmm yeah you're right. I was thinking we had a list of `artistIds` and then used that to fetch their full `User`s but that's actually all happening inside the selector so this is fine I think

brain might be stuck on the suggested follows bugfix I was doing last week 😅